--- a/torch/__init__.py	2026-04-29 14:33:55.570908522 +0200
+++ b/torch/__init__.py	2026-04-29 14:33:55.586908713 +0200
@@ -308,30 +308,7 @@
     here = os.path.abspath(__file__)
     global_deps_lib_path = os.path.join(os.path.dirname(here), "lib", lib_name)
 
-    try:
-        ctypes.CDLL(global_deps_lib_path, mode=ctypes.RTLD_GLOBAL)
-        # Workaround slim-wheel CUDA dependency bugs in cusparse and cudnn by preloading nvjitlink
-        # and nvrtc. In CUDA-12.4+ cusparse depends on nvjitlink, but does not have rpath when
-        # shipped as wheel, which results in OS picking wrong/older version of nvjitlink library
-        # if `LD_LIBRARY_PATH` is defined, see https://github.com/pytorch/pytorch/issues/138460
-        # Similar issue exist in cudnn that dynamically loads nvrtc, unaware of its relative path.
-        # See https://github.com/pytorch/pytorch/issues/145580
-        try:
-            with open("/proc/self/maps") as f:
-                _maps = f.read()
-            # libtorch_global_deps.so always depends in cudart, check if its installed via wheel
-            if "nvidia/cuda_runtime/lib/libcudart.so" not in _maps:
-                return
-            # If all above-mentioned conditions are met, preload nvrtc and nvjitlink
-            # Please note that order are important for CUDA-11.8 , as nvjitlink does not exist there
-            _preload_cuda_deps("cuda_nvrtc", "libnvrtc.so.*[0-9]")
-            _preload_cuda_deps("nvjitlink", "libnvJitLink.so.*[0-9]")
-        except Exception:
-            pass
-
-    except OSError as err:
-        # Can only happen for wheel with cuda libs as PYPI deps
-        # As PyTorch is not purelib, but nvidia-*-cu12 is
+    if True:
         from torch.version import cuda as cuda_version
 
         cuda_libs: dict[str, str] = {
@@ -357,11 +334,6 @@
             if t_major >= 12:
                 cuda_libs["cufile"] = "libcufile.so.*[0-9]"
 
-        is_cuda_lib_err = [
-            lib for lib in cuda_libs.values() if lib.split(".")[0] in err.args[0]
-        ]
-        if not is_cuda_lib_err:
-            raise err
         for lib_folder, lib_name in cuda_libs.items():
             _preload_cuda_deps(lib_folder, lib_name)
         ctypes.CDLL(global_deps_lib_path, mode=ctypes.RTLD_GLOBAL)
